strictEqual.js

```diff
@@ -0,0 +1,9 @@
+const strictEqual = (a, b) => {
+
+    if (typeof (a) == typeof (b) && a == b) {
+        return true
+    }
+    return false
```

Comment:
Here we're saying if the condition is true return true, if the condition is false, return false. That means we can just return the condition since it will be the same as the value we want returned.

```JavaScript
const strictEqual = (val1, val2) => {
    return typeof val1 == typeof val2 && val1 == val2
}
```

Doing that makes this a one line return function which means we can rewrite it using fat arrow's implicit return.

```JavaScript
const strictEqual = (val1, val2) => typeof val1 == typeof val2 && val1 == val2
```